### PR TITLE
docs(reproduce): add Issue #49 VisDrone and SKU-110K results

### PR DESCRIPTION
**Pull request by 465114810-debug**
## Summary

Related to #49.

This PR adds reproducible 100-epoch results for:

- YOLO-Master-v0.1-N on VisDrone
- YOLO-Master-EsMoE-N on VisDrone
- YOLO-Master-v0.1-N on SKU-110K
- YOLO-Master-EsMoE-N on SKU-110K

It also publishes the existing `mixture_aux_loss` under the canonical W&B metric name `moe_loss`, while preserving the original metric name.

## Changes

- Add canonical W&B aliases:
  - `train/moe_loss`
  - `val/moe_loss`
- Preserve:
  - `train/mixture_aux_loss`
  - `val/mixture_aux_loss`
- Add VisDrone result files:
  - `scripts/reproduce/results/issue49_465114810_visdrone.csv`
  - `scripts/reproduce/results/issue49_465114810_visdrone.md`
- Add SKU-110K result files:
  - `scripts/reproduce/results/issue49_465114810_sku110k.csv`
  - `scripts/reproduce/results/issue49_465114810_sku110k.md`

## Training configuration

| Item | Value |
|---|---|
| GPU | NVIDIA GeForce RTX 4090 D, 24 GB |
| Epochs | 100 |
| Image size | 640 |
| Batch size | 32 |
| Workers | 8 |
| Seed | 42 |
| Pretrained weights | Disabled |
| Optimizer | `auto` |
| W&B project | `yolo-master-issue49-465114810` |

## Results

Best epoch is selected by maximum validation `mAP50-95`.

| Dataset | Model | Evaluation | Best epoch | Precision | Recall | mAP50 | mAP50-95 | Hours |
|---|---|---|---:|---:|---:|---:|---:|---:|
| VisDrone | v0.1-N | standard | 88 | 0.43025 | 0.33008 | 0.30384 | 0.17009 | 2.178 |
| VisDrone | EsMoE-N | dense | 89 | 0.43094 | 0.33943 | 0.30861 | 0.17344 | 2.962 |
| SKU-110K | v0.1-N | standard | 91 | 0.90319 | 0.83643 | 0.88250 | 0.54279 | 3.275 |
| SKU-110K | EsMoE-N | dense | 93 | 0.90764 | 0.83626 | 0.88434 | 0.54519 | 3.301 |

### VisDrone

- v0.1-N: https://wandb.ai/465114810-ab/yolo-master-issue49-465114810/runs/o2ur8w66
- EsMoE-N: https://wandb.ai/465114810-ab/yolo-master-issue49-465114810/runs/6qd0dn71

### SKU-110K

- v0.1-N: https://wandb.ai/465114810-ab/yolo-master-issue49-465114810/runs/68e3g0lf
- EsMoE-N: https://wandb.ai/465114810-ab/yolo-master-issue49-465114810/runs/674mjnj5

All links were verified in a logged-out private browser window.

## EsMoE validation

EsMoE-N was evaluated with:

```bash
--no-sparse-eval

### CODE REVIEW

**Comment by isLinXu** [comment]
LGTM
